docker/Dockerfile

```diff
@@ -30,7 +30,7 @@ RUN rm /bin/sh && ln -s /bin/bash /bin/sh
 WORKDIR /opt/openoni
 ADD . /opt/openoni
 
-ADD entrypoint.sh /
+RUN cp /opt/openoni/docker/entrypoint.sh /entrypoint.sh
```

Comment:
Can you comment on this being copied via command rather than added as before?  I'm not sure why this was changed.